legacy/README.md

```diff
@@ -1,4 +1,17 @@
-# FairSoft
+# FairSoft (Legacy)
+
+## Preface
+
+This will be the last release featuring the bash/cmake
+based setup system. Upcoming releases are planned to use
+spack instead (for more information see
+[here](../docs/README.md)).
+
+The remainder of this README is basically the main README
+from the last releases. All file references should be read
+as relative to the `/legacy/` folder.
```

Comment:
```suggestion
Our homegrown bash/cmake based setup system
has been named "Legacy". It has been moved to the
sub-directory `legacy/` to distinguish it clearly
from the future Spack-based setup system
(for more information see [here](../docs/README.md)).
The latter will eventually replace the "Legacy" setup system
in a future release.
```

* I prefer to advertize the upcoming spack-only release "less final". Sure, right now it looks like it is going to be the next release in half a year. But what if we have to release another time in between, which we do not know yet today.
* Let's delete the note on references and rather update such references everywhere please.